modules/iam/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.0.0, < 2.0.0"

```

Comment:
Let's bump a Terraform version in refactor. I propose required_version = ">= 1.5.0, < 2.0.0"